src/pages/components/text-input/usage.mdx

```diff
@@ -456,39 +550,6 @@ characters allowed or date structure.
 </Column>
 </Row>
 
-## Text area
-
-Use a text area when the expected user input is more than a few words and could
-span multiple lines. It is commonly used for features like user commentary or
-descriptions. It supports all the same states and functionality as text input
-except for the password functionality. Text area has several unique
-functionalities not included in the default text input, like the resize handle
-and character counter.
-
-<Row>
-<Column colLg={8}>
-
-![Example of a text area](images/text-input-usage-2.png)
-
-</Column>
-</Row>
-
-### Resize handle
-
-Included by default in text area is the `resize` handle. It allows a user to
-manipulate the field height by making it longer or shorter. The resize handle
-has no effect on the width of the text area container, it only effects the
-height. If the user makes the field size shorter than the content inside the
-field then a vertical scroll will become available.
-
-<Row>
-<Column colLg={8}>
-
-![Example showing the resize handle in text area](images/text-area-usage-resize.png)
-
-</Column>
-</Row>
-
 ### Character counter
```

Comment:
The image for the Character counter has a typo. Charge should be `Charged`

Reply:
dang, you caught that typo last time too, how did it slip in again 😆

Reply:
Ok this is corrected in the source image but the preview image isn't updating to show change.